### ISSUE TITLE
Docker freezes despite creating a new container

### ISSUE THREAD

**Issue by mavisczx** [nus-cs2030/2223-s2#178]
# Summary
When I open docker in the terminal, compile and open jshell a few times, my terminal becomes progressively slower,  in compiling (javac) and opening jshell until it ultimately lags permanently and I am unable to execute any commands (shown in picture).

## Description 
At first I thought that creating a new container, copying my files to the new container will rectify the problem, but the same problem occurred. I have to stop running the docker, run the docker again, open the terminal before the problem is solved (momentarily), before the same issue arises. 
Is anybody facing the same issue and knows what is the problem?
Thank you very much! 
(for context laptop runs on M1 pro chip) 

## Screenshots (if any): 
![image](https://user-images.githubusercontent.com/122339026/219925969-52bd70c2-b115-445c-afb2-72fa6a866bf1.png)

**Comment by aowz** [comment]
Might sound stupid but shutting down my computer and restarting the whole process helped. Not sure why this helps but it does. Hope it works for you!

**Comment by stxe09** [comment]
yup i had this issue too and I just shut down everything and restarted. i think it happens when you leave docker running too long/ haven't restarted your laptop in ages
